src/app/frontend/common/filters/relativetime_filter.js

```diff
@@ -92,11 +97,28 @@ export default function relativeTimeFilter() {
       return formatOutputTimeString_(diffInYears, Units.YEAR);
     }
   };
+
   return filterFunction;
 }
 
 /**
+ * Returns current time. If appConfig.serverTime is provided then it will be returned, otherwise current
+ * client time will be used.
+ *
+ * @return {number}
+ * @private
+ */
+function getCurrentTime() {
+  if (appConfig && appConfig.serverTime) {
```

Comment:
@bryk I've issue with making this line pass `gulp-closure-compiler` check. Could you take a look at it? Do you have any suggestions?